### PR TITLE
feat(24.04): add openjdk-17-jre-headless dependencies

### PR DESCRIPTION
**Pull request by vpa1977**
# Proposed changes

Copy openjdk 17 dependencies slice definitions from 22.04 as they are missing from 24.04.

## Related issues/PRs
Depends on:
https://github.com/canonical/chisel-releases/pull/308

## Checklist

* [x] I have read the [contributing guidelines](
https://github.com/canonical/chisel-releases/blob/main/CONTRIBUTING.md)
* [x] I have tested my changes ([see how](https://github.com/canonical/chisel-releases/blob/main/CONTRIBUTING.md#7-test-your-slices-before-opening-a-pr))
* [x] I have already submitted the [CLA form](
https://ubuntu.com/legal/contributors/agreement)

### CODE REVIEW

**Review by cjdcordeiro — COMMENTED** [review]
To be reviewed once #308 is in

File: slices/libglib2.0-0t64.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package: libglib2.0-0t64
+
+essential:
+  - libglib2.0-0t64_copyright
+
+slices:
+  core:
```
Comment (cjdcordeiro) [inline slices/libglib2.0-0t64.yaml:7]: interesting - first use of a core slice for a libs package :)

**Review by cjdcordeiro — COMMENTED** [review]
lgtm thanks

**Review by rebornplusplus — APPROVED** [review]
Nicely done!

File: slices/libglib2.0-0t64.yaml (hunk 1 above)
Comment (vpa1977) [reply slices/libglib2.0-0t64.yaml:7]: I had to split this one as core is needed for java.base, but the rest is only relevant for awt =)

**Review by cjdcordeiro — APPROVED** [review]

**Pull request merged**